### ISSUE TITLE
v2.10.0

### ISSUE THREAD

**Issue by angad777** [prisma/migrate#627]
Hi Prisma Team! Prisma Migrate just crashed. 

## Versions
      
| Name        | Version            |
|-------------|--------------------|
| Platform    | darwin             | 
| Node        | v12.18.0           | 
| Prisma CLI  | 2.10.0             | 
| Binary      | af1ae11a423edfb5d24092a85be11fa77c5e499c| 

## Error
```
Error: Error in migration engine.
Reason: [libs/sql-schema-describer/src/walkers.rs:214:27] index out of bounds: the len is 0 but the index is 0

Please create an issue in the migrate repo with
your `schema.prisma` and the prisma command you tried to use 🙏:
https://github.com/prisma/migrate/issues/new

```

**Comment by tomhoule** [comment]
We worked on this problem and just released a fix in 2.10.1. Can you confirm that this does not happen anymore with the new version?

**Comment by angad777** [comment]
Hey @tomhoule - I can confirm the above issue is now resolved in v2.10.1.

Appreciate the prompt fix.

Cheers
